tests/doorbell_traits_test.py

```diff
@@ -11,22 +11,24 @@ def test_doorbell_chime():
     device = Device.MakeDevice(raw, auth=None)
     assert "sdm.devices.traits.DoorbellChime" in device.traits
 
+
 def test_doorbell_chime_trait_hack():
     """Adds the DoorbellChime trait even when missing from the API to fix an API bug."""
     raw = {
         "name": "my/device/name",
         "type": "sdm.devices.types.DOORBELL",
-        "traits": { },
+        "traits": {},
```

Comment:
To double check: Are these from running these commands?
https://github.com/allenporter/python-google-nest-sdm#development

Its not automated with a hook or anything.  I don't know if the formatting change you made is from running it because I forgot, but just thought I would ask to make sure.  that would make sense because this doorbell change was the last one i made.

Reply:
Yeah, I ran the 3 commands prior to checkin.